### ISSUE TITLE
Make capstone checkout and build optional

### ISSUE THREAD

**Issue by radare** [radare/radare2#4044]
Something necessary when github is down or you are alergic to bsd

**Comment by snare** [comment]
I think it'd be nice to `git clone --depth 1 ...` capstone if possible. I'm working on r2 support for voltron ATM and I spend a lot of time destroying and re-provisioning Vagrant VMs on a slow connection :(

**Comment by radare** [comment]
yeah :( capstone is a pretty huge dependency. i was using —depth 1 in all the other external repos, but, as long as we have to apply patches on top of that and clone a specific commit the build fails if we use —depth 1.

i can try to find a faster solution, working offline use to be a pain just because of capstone.. so you can also use —with-sys-capstone and just use the version you have installed in the system, which would be faster than cloning+building everytime

**Comment by snare** [comment]
Ahhh yeah that is annoying. I might try installing capstone in my Vagrantfile and using `--with-sys-capstone`. It's not a huge problem, I just noticed it was one of the longer delays in provisioning my Vagrant VM :)

**Comment by XVilka** [comment]
@radare can be closed now I guess?

**Comment by radare** [comment]
Cs is not an optonal dep for many important archs right now
